Dockerfile.php5.6

```diff
@@ -1,91 +1,78 @@
-FROM php:5.6-fpm-alpine
-
-RUN apk add --no-cache \
-		bash \
-		sed
+FROM php:5.6-fpm
 
 RUN set -ex; \
 	\
-	apk add --no-cache --virtual .build-deps \
+ 	apt-get update && apt-get install -y --no-install-recommends \
 		autoconf \
-		build-base \
-		bzip2-dev \
-		gettext-dev \
-		icu-dev \
-		imagemagick-dev \
-		imap-dev \
-		libjpeg-turbo-dev \
+		build-essential \
+		gettext \
+		libbz2-dev \
+		libicu-dev \
+		libmagick++-dev \
+		libjpeg62-turbo-dev \
 		libmcrypt-dev \
 		libmemcached-dev \
-		libpng-dev \
-		libssh2-dev \
+		libssh2-1-dev \
 		libtool \
-		libxml2-dev \
-		libxslt-dev \
-		openldap-dev \
-		pcre-dev \
-	; \
-	docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr; \
-	docker-php-ext-install \
-		bcmath \	
-		bz2 \
-		calendar \
-		dba \
-		exif \
-		gd \
-		gettext \
-		imap \
-		intl \
-		ldap \
-		mcrypt \
+		libxslt1-dev \
+		libldap2-dev \
+		libc-client2007e-dev \
+		libkrb5-dev \
+		pax-utils \
+ 	; \
+ 	docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr; \
+	docker-php-ext-configure imap --with-kerberos --with-imap-ssl; \
+	docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/; \
+ 	docker-php-ext-install \
+ 		bcmath \
+ 		bz2 \
+ 		calendar \
+ 		dba \
+ 		exif \
+ 		gd \
+ 		gettext \
+ 		imap \
+ 		intl \
+ 		ldap \
+ 	 	mcrypt \
 		mysql \
-		mysqli \
-		opcache \
-		pdo_mysql \
-		shmop \
-		soap \
-		sockets \
-		sysvmsg \
-		sysvsem \
-		sysvshm \
-		wddx \
-		xmlrpc \
-		xsl \ 
-		zip \
-	; \
-	pecl install \
-		igbinary \
-		imagick \
-		memcached-2.2.0 \
-		msgpack-0.5.7 \
-		redis \
-	; \
-	echo "\n" | pecl install ssh2-0.13; \
-	docker-php-ext-enable --ini-name pecl.ini \
-		igbinary \
-		imagick \
-		memcached \
-		msgpack \
-		redis \
-		ssh2 \
-	; \
-	runDeps="$( \
-		scanelf --needed --nobanner --recursive \
-			/usr/local/lib/php/extensions \
-			| awk '{ gsub(/,/, "\nso:", $2); print "so:" $2 }' \
-			| sort -u \
-			| xargs -r apk info --installed \
-			| sort -u \
-	)"; \
-	apk add --virtual .wordpress-phpexts-rundeps $runDeps; \
-	apk del .build-deps; \
```

Comment:
Do we have to install the non-dev version independently so that they don't get removed along with the -dev packages?